README.md

```diff
@@ -85,3 +101,5 @@ Consult also [our demo sample](samples/iframe_mobile.html) which shows the simpl
 ## Desktop: display a QR code
 
 On the product page from which you would like the client to start virtual try-on, display a QR code that the client can scan with their phone and go directly to Virtual Fitting Room on mobile. [Our sample](samples/desktop.html) uses the open-source [QR-Code-generator](https://github.com/nayuki/QR-Code-generator) library for demo purposes. Choose any QR code generating tool that suits you.
+
+For the 3D Viewer component, integrate the iframe in the same way as you would [on mobile](#mobile-show-the-fitting-room). The only difference is be that you set the `viewmode` query parameter to `desktop` instead of `mobile`.
```

Comment:
Can we add here sample images from the slide https://docs.google.com/presentation/d/1DMPs6F33LOCvE0twhrVQXIcGRTf_4LGKoJl0pvDjG8k/edit#slide=id.g1fcb4a2b870_0_25

Reply:
Done, converting to a regular PR and waiting for the final review.